BlindspotAssistance/common/eis_common/libs/EISMessageBus/examples/pub_thread.cpp

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) 2019 Intel Corporation.
```

Comment:
2020, if the date represents the last year edition

Reply:
example code, not related to BlindSpot execution.

Reply:
fixed